apps/nextjs-website/src/_contents/send/guides.ts

```diff
@@ -55,6 +55,10 @@ const knowledgeBase: GuideDefinition = {
       version: 'v2.0',
       dirName: 'LkozJ77TMlb5XupVDO7J',
     },
+    {
+      version: 'v2.1',
+      dirName: 'LjuHmRnHyge98hRa4Ao4',
+    },
```

Comment:
Why it is not in the position 0? I think the version menu on the guide page maintains this order

Reply:
because 2.0, temporarily will remain the main version but some users are already using version 2.1. It's dev Vs prod situation.